lib/src/main/java/graphql/nadel/engine/transform/hydration/batch/NadelBatchHydrationInputBuilder.kt

```diff
@@ -54,8 +55,19 @@ internal object NadelBatchHydrationInputBuilder {
                     }
                     // These are batch values, ignore them
                     is NadelHydrationActorInputDef.ValueSource.FieldResultValue -> null
+                    is NadelHydrationActorInputDef.ValueSource.StaticValue -> {
+                        val staticValue: NormalizedInputValue? = makeNormalizedInputValue(
+                            type = actorFieldArg.actorArgumentDef.type,
+                            value = valueSource.value,
+                        )
+                        if (staticValue != null) {
+                            actorFieldArg to staticValue
+                        } else {
+                            null
+                        }
```

Comment:
if it ended up null - should it be a map of name to null?

Why does this `makeNormalizedInputValue` methods return null is really the question - is it a I could not handle this

like can you do ` [{name: "someAttribute" value: null}]` say ??

Reply:
So looking into it, this function doesn't actually return null. This syntax was just a result of copying the `ArgumentValue` case above. I will update now

Reply:
> Why does this makeNormalizedInputValue methods return null is really the question

so the answer is actually it doesn't